src/towncrier/test/test_create.py

```diff
@@ -218,16 +414,18 @@ def test_create_orphan_fragment(self, runner: CliRunner):
         self.assertEqual(2, len(fragments))
         change1, change2 = fragments
 
-        self.assertEqual(change1.suffix, ".feature")
+        self.assertEqual(change1.suffix, ".rst")
```

Comment:
It looks like the expectation was changed here, without any changes in the test setup.

Is this a change that modified the default behaviour ?

I think that is ok to change the default behaviour, but this needs to be mentioned in the release notes.

This project is all about release notes, so I think it's important for `towncrier` to be a good example of how the release notes should be managed :)

Reply:
Now mentioned in release notes